<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Считать граф</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Матрица расстояний</string>
   </property>
  </widget>
  <widget class="QTableView" name="T_table">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>461</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>30</y>
     <width>121</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Список смежности</string>
   </property>
  </widget>
  <widget class="QTableView" name="Sm_tableView">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Матрица смежности</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_FROM">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>310</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>320</y>
     <width>41</width>
     <height>57</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Куда:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>59</width>
     <height>57</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Откуда:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_TO">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>340</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="long_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>460</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="way">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>390</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_PTY">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>370</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Gothic</family>
    </font>
   </property>
   <property name="text">
    <string>путь</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>300</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество маршрутов</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>330</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>330</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Длиной=</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>360</y>
     <width>161</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>460</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Длиной=</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
